src/odin/maneuver.cc

```diff
@@ -1119,6 +1119,14 @@ void Maneuver::set_long_street_name(bool has_long_street_name) {
   has_long_street_name_ = has_long_street_name;
 }
 
+BikeShareStationInfo Maneuver::bss_info() const {
```

Comment:
Did you want to return by value or const ref?

Reply:
My bad I’ll correct it !